lib/vmpooler/metrics/promstats.rb

```diff
@@ -0,0 +1,299 @@
+# frozen_string_literal: true
+
+module Vmpooler
+  class Promstats < Metrics
+    attr_reader :prefix, :endpoint, :metrics_prefix
+
+    # Constants for Metric Types
+    M_COUNTER   = 1
+    M_GAUGE     = 2
+    M_SUMMARY   = 3
+    M_HISTOGRAM = 4
+
+    # Customised Bucket set to use for the Pooler clone times set to more appropriate intervals.
+    POOLER_TIME_BUCKETS = [1.0, 2.5, 5.0, 10.0, 20.0, 50.0, 100.0, 200.0, 500.0, 1000.0, 2000.0].freeze
```

Comment:
@mattkirby Can you take a look at this and the REDIS connection buckets and see if you want any changes.

Reply:
Hmm @mattkirby